clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThread.java

```diff
@@ -154,16 +153,12 @@ void runOnce() {
         lastPollTimeMs = currentTimeMs;
 
         final long pollWaitTimeMs = requestManagers.entries().stream()
-                .filter(Optional::isPresent)
-                .map(Optional::get)
                 .map(rm -> rm.poll(currentTimeMs))
                 .map(networkClientDelegate::addAll)
                 .reduce(MAX_POLL_TIMEOUT_MS, Math::min);
         networkClientDelegate.poll(pollWaitTimeMs, currentTimeMs);
 
         cachedMaximumTimeToWait = requestManagers.entries().stream()
-                .filter(Optional::isPresent)
-                .map(Optional::get)
                 .map(rm -> rm.maximumTimeToWait(currentTimeMs))
                 .reduce(Long.MAX_VALUE, Math::min);
```

Comment:
it seems to me `reduce` is not straightforward. maybe we can use `min` instead.
```java
                .mapToLong(rm -> rm.maximumTimeToWait(currentTimeMs))
                .min()
                .orElse(Long.MAX_VALUE);
```

Reply:
Done.